multi-arch-builders/coreos-ppc64le-builder-512e.bu

```diff
@@ -0,0 +1,38 @@
+# This butane config will do the following:
```

Comment:
And maybe here we should specify that it's intended for systems with 512e disks?